<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyncGateEmbeddedWidget</class>
 <widget class="QWidget" name="SyncGateEmbeddedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>158</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="mpIn0Checkbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Bool</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mpOut0Checkbox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Bool</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mpIn1Checkbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Bool</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mpOut1Checkbox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Bool</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
